src/quill-editor.component.ts

```diff
@@ -235,7 +241,7 @@ export class QuillEditorComponent
 
   ngOnChanges(changes: SimpleChanges): void {
     if (!this.quillEditor) {
-      return null;
+      return;
```

Comment:
This minor change just fixes a type issue (`ngOnChanges` is declared with a `void` return yet it was retuning `null`).